### PR TITLE
Add swift3 to Dockerfile

### PR DESCRIPTION
**Pull request by andrewgaul**
This allows testing with clients like s3cmd and s3fs.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -34,6 +37,7 @@ RUN	chmod +x /swift/bin/*
 ADD	./misc/bashrc /swift/.bashrc
 
 RUN	cp /usr/local/src/swift/test/sample.conf /etc/swift/test.conf
+RUN	sed -i 's/pipeline = healthcheck proxy-logging cache tempauth proxy-logging proxy-server$/pipeline = healthcheck proxy-logging cache swift3 tempauth proxy-logging proxy-server/' /etc/swift/proxy-server.conf ; echo "\n[filter:swift3]\nuse = egg:swift3#swift3" >> /etc/swift/proxy-server.conf
```
Comment (kahing) [inline Dockerfile]: just modify the proxy config instead of using sed?

**Comment by pbinkley** [comment]
I won't be able to put any time into this project going forward. Would one or both of you care to take it over? If so, I think it would be easiest to have you fork it, and I'll change the readme here to point to your fork as the current version.

**Comment by kahing** [comment]
I can volunteer and I already have a fork. Ideally you could transfer the repo to me and then github would do all the redirects correctly.

**Comment by andrewgaul** [comment]
@kahing is in a better position than me to maintain docker-swift.

**Comment by pbinkley** [comment]
OK, I'll transfer ownership to @kahing .

**Comment by pbinkley** [comment]
Argh, it fails with "kahing/docker-swift already exists". Do you want to delete your fork, accept the transfer, and then update from local? Let me know how you'd like to proceed.

**Comment by kahing** [comment]
I've deleted my repository

**Comment by pbinkley** [comment]
It's on its way.

Peter Binkley, Ph.D., MLIS
Digital Initiatives Technology Librarian
Information Technology Services
peter.binkley@ualberta.ca

2-10K Cameron Library
University of Alberta
Edmonton, Alberta
Canada T6G 2J8

phone 780-492-3743
fax 780-492-9243

File: Dockerfile (hunk 1 above)
Comment (andrewgaul) [inline Dockerfile]: Done.

File: Dockerfile [hunk 2]
```diff
@@ -8,13 +8,15 @@ RUN	dpkg-divert --local --rename --add /sbin/initctl; ln -sf /bin/true /sbin/ini
 
 RUN	apt-get update; apt-get upgrade -y
 
-RUN	apt-get install -y rsyslog; apt-get install -y curl gcc memcached rsync sqlite3 xfsprogs git-core libffi-dev python-setuptools sudo python-coverage python-dev python-nose python-simplejson python-xattr python-eventlet python-greenlet python-pastedeploy python-netifaces python-pip python-dnspython python-mock sysklogd attr openssh-server openssh-client
+RUN	apt-get install -y rsyslog; apt-get install -y curl gcc memcached rsync sqlite3 xfsprogs git-core libffi-dev python-setuptools sudo python-coverage python-dev python-nose python-simplejson python-xattr python-eventlet python-greenlet python-pastedeploy python-netifaces python-pip python-dnspython python-mock sysklogd attr openssh-server openssh-client \
+    python-lxml
 
 RUN	cd /usr/local/src; git clone --depth 1 https://github.com/openstack/python-swiftclient.git
 RUN	cd /usr/local/src; git clone --depth 1 https://github.com/openstack/swift.git
```
Comment (andrewgaul) [inline Dockerfile]: I lost the git clone during rebasing, thanks @shooding!

**Pull request closed without merging**